### DISCUSSION TITLE
big endian and little endian

### DISCUSSION THREAD

**Discussion by sajvn0x** [obhq/exfat#6] [General]
Why Are Big Endian and Little Endian Byte Ordering Is Important To This Project (Depending on byteorder crate), Can You give me a brief explanation, or give me some links to why these things important in programming??

**Comment by ultimaweapon** [comment]
You can see in the specs https://learn.microsoft.com/en-us/windows/win32/fileio/exfat-specification#14-default-field-and-structure-qualifiers that most data in exFAT use little-endian.
